src/docs/product/sentry-basics/guides/integrate-frontend/create-new-project.mdx

```diff
@@ -3,72 +3,71 @@ title: Create a Sentry Project
 
-   - Give the project a `Name`.
+   - Give the project a name in the "Project name" field.
 
-   - Under `Set your default alert settings` check `i'll create my own alerts later`
+   - Under "Set your default alert settings", select "I'll create my own alerts later".
 
-   - `Assign a Team` to the project.
+   - Assign a "Team" to the project.
 
      ![Select project language](create-new-project-02.png)
 
-     > If you haven't defined any teams, you can either select the default org team (the team that has the same name as your Sentry Org) or click on the `+` button to create a new team.
+     > If you haven't defined any teams, you can either select the default org team (the team that has the same name as your Sentry org) or click on the "+" button to create a new team.
 
      ![Select team](create-new-project-03.png)
 
-   - Click on `Create Project`.
-     This takes you to the configuration page. Read through the quick Getting Started guide.
+   - Click "Create Project". This takes you to the configuration page. Read through the quick Getting Started guide.
 
-4. Copy the `DSN key` and keep it handy as we will be copying the key into the source code.
+4. Copy the "DSN" key and keep it handy as you will be copying the key into the source code.
 
    ![Copy DSN](create-new-project-04.png)
 
    > The DSN (or Data Source Name) tells the SDK where to send the events, associating them with the project you just created.
 
-5. Click on the `Got it!` button at the bottom to create the project.
+5. Click on the "Got it!" button at the bottom to create the project.
 
 ## Step 2: Create an alert rule
 
-You can create various alert rules per project and let Sentry know when, how, and whom you want to notify when errors occur in your application. **Alert rules** consist of **Conditions** and **Actions**, which are performed when the associated conditions are met. For more information, see [Alerts](/product/alerts-notifications/alerts/). When creating a new project, you can select to create it with an alert rule that notifies all project team members (via email) the first time a new **issue** appears. This means that the next time a **similar error** occurs, the notification is not triggered since the error is not "new."
+You can create various alert rules for each project and let Sentry know when, how, and whom you want to notify when errors occur in your application. *Alert rules* consist of *conditions* and *actions*, which are performed when the associated conditions are met. Learn more in our [Alerts documentation](/product/alerts-notifications/alerts/). When creating a new project, you can choose to create it with an alert rule that notifies all project team members (by email) the first time a new *issue* appears. This means that the next time a **similar error** occurs, the notification is not triggered since the error is not "new".
```

Comment:
```suggestion
You can create various [alert rules](/product/alerts-notifications/alerts/) for each project and let Sentry know when, how, and whom you want to notify when errors occur in your application. *Alert rules* consist of *conditions* and *actions*, which are performed when the associated conditions are met. When creating a new project, you can choose to create it with an alert rule that notifies all project team members (by email) the first time a new *issue* appears. This means that the next time a **similar error** occurs, the notification is not triggered since the error is not "new".
```